### ISSUE TITLE
About example

### ISSUE THREAD

**Issue by chaosicong** [omoultosEthTuDelft/OCTP#20]
I tried running the example you provided, but it got an error at line 113，I'm curious as to why this is happening, because there was also such an command before line 113.
#
![屏幕截图 2024-03-19 214251](https://github.com/omoultosEthTuDelft/OCTP/assets/147289858/f1b4df39-b59e-4c74-a6eb-3a7b88514f76)

**Comment by omoultosEthTuDelft** [comment]
this is because of the new version of LAMMPS. Remove the reset_timestep command.

**Comment by chaosicong** [comment]
Yes, I've successfully fixed this issue, but I still have a problem.In the following picture, you increase the temperature and then decrease the pressure, and finally balance at the target temperature and pressure. Why not balance directly at the target pressure and temperature? If it is because of annealing, why not increase and decrease the temperature at NVT? I see that you also increased and decreased the pressure. What is the meaning of doing so? I really don't understand. However, it does seem unrelated to OCTP, and I do want to understand the reasons for learning. thanks
![屏幕截图 2024-03-21 215339](https://github.com/omoultosEthTuDelft/OCTP/assets/147289858/c6984458-283e-46c2-aa7a-48dd17e337e6)

**Comment by chaosicong** [comment]
used the average volume, but  used the position of the last step of the atom, whether this is reasonable, I sometimes get an error when I run it

**Comment by omoultosEthTuDelft** [comment]
this is a scheme for equilibrating the system. in this way you are safe from overlaps that have possibly occured in the construction of the initial configuration. it is unrelated to OCTP. For more details on this please read our relevant publications.

**Comment by chaosicong** [comment]
Why does the supplementary material of the paper say that first NPT, then NVT, and finally NVE. However, the full example in "Complete example with postprocessing in python' provided only uses NPT and NVT.

**Comment by omoultosEthTuDelft** [comment]
it does not matter. OCTP is a tool that does not depend on the statistical ensemble. what we do in the paper is to ensure that molecular simulations are properly performed. this github is about OCTP, not running LAMMPS in general, or to teach mol sim.
